releases/net471/KnownIssues/514195-Targeting .NET Framework 4.7.1 will copy extra files to your bin directory.md

```diff
@@ -0,0 +1,24 @@
+# Targeting .NET Framework 4.7.1 will copy extra files to your bin directory
+
+## Symptoms
+
+When building an application that targets .NET Framework 4.7.1, the tooling will copy 12 assemblies into your bin directory if you consume one or more .NET Standard-based components.
```

Comment:
Let's be explicit about which version of VS will add those files.

Reply:
"the tooling" is somewhat vague, although perhaps OK. But Alex's comment about the version of VS is significant. When does this happen? In VS only, when building from the command-line using MSBuild (so it's an MSBuild issue), or with a command-line compiler? So instead of the tooling, "Visual Studio \<version> copies..."

will copy --> copies (almost always best to use present tense instead of future)